yt_dlp/extractor/jupiter.py

```diff
@@ -0,0 +1,59 @@
+from .common import InfoExtractor
+from .. import int_or_none
+from ..utils import (
+    traverse_obj,
+)
```

Comment:
```suggestion
from .common import InfoExtractor
from ..utils import clean_html, int_or_none, str_or_none, url_or_none
from ..utils.traversal import traverse_obj
```

Reply:
yeah. this is auto-filled by ide (pycharm). is there a formatter that would sort them?

Reply:
I've auto-fixed with [isort](https://pycqa.github.io/isort/): 3c5641301